### PR TITLE
fix #79 Goal Sabotage is broken

### LINKED ISSUE #79: Goal Sabotage is broken
- Ignite TNT does not make team lose/win.
- No protection againt teamate igniting there own tnt
- TNTs are note replaced after battle. (if we don't use blockrestore and WE of course)

### PR DESCRIPTION
**Pull request by Oruss7**
Version PRE 2.0.

A lot of copied/pasted code...

### CODE REVIEW

File: src/net/slipcor/pvparena/core/Language.java [hunk 1]
```diff
@@ -531,8 +531,9 @@ private Language() {
         GOAL_SABOTAGE_IGNITED("nulang.goal.sabotage.tntignite", "%1% ignited the TNT of team %2%!"),
         GOAL_SABOTAGE_SETTNT("nulang.goal.sabotage.set", "TNT set: %1%"),
         GOAL_SABOTAGE_TOSETTNT("nulang.goal.sabotage.toset", "TNT to set: %1%"),
-        GOAL_SABOTAGE_YOUCANNOTSELFDESTROY("nulang.goal.sabotage.youcannotselfdestroy", "You can not ignite your own TNT!'"),
-        GOAL_SABOTAGE_YOUTNT("nulang.goal.sabotage.youtnt", "You now carry the sabotage materials!'"),
+        GOAL_SABOTAGE_YOUCANNOTSELFDESTROY("nulang.goal.sabotage.youcannotselfdestroy", "You can not ignite your own TNT!"),
```
Comment (Eredrim) [inline src/net/slipcor/pvparena/core/Language.java:534]: I suggest to simplify with `GOAL_SABOTAGE_NOSELFDESTROY` and `nulang.goal.sabotage.noselfdestroy`

File: lang/lang_es-es.yml [hunk 2]
```diff
@@ -248,7 +248,9 @@ nulang:
       tntignite: '%1% encendió la TNT del Equipo %2%!'
       set: 'TNT añadida: %1%'
       toset: 'TNT por añadir: %1%'
-      youtnt: ¡Estas llevando los materiales de Sabotage!'
+      youcannotselfdestroy: 'You can not ignite your own TNT!'
+      youtnt: '¡Estas llevando los materiales de Sabotage!'
+      notgooditem: 'You need sabotage tool to ignite the TNT.'
```
Comment (Eredrim) [inline lang/lang_es-es.yml:253]: You don't need to add english sentences to other languages. If language key is not found, default value (in english) from Language.java will be used.

**Review by Eredrim — CHANGES_REQUESTED** [review]

File: src/net/slipcor/pvparena/goals/GoalSabotage.java [hunk 3]
```diff
@@ -555,24 +581,19 @@ public void onTNTExplode(final EntityExplodeEvent event) {
...
 
-            if (nearest.getDistanceSquared(tLoc) < 4) {
-                event.setCancelled(true);
-                return;
+                commit(this.arena, team.getName());
+                World world = event.getEntity().getLocation().getWorld();
+                Location location = event.getEntity().getLocation();
+                tnt.remove();
+                world.spawnParticle(Particle.EXPLOSION_LARGE, location.getX(), location.getY() + 1, location.getZ(), 25);
+                world.playSound(location, Sound.ENTITY_GENERIC_EXPLODE, 20, 2);
```
Comment (Eredrim) [inline src/net/slipcor/pvparena/goals/GoalSabotage.java:593]: Really cool, this fake explosion!

**Review by Eredrim — APPROVED** [review]

File: src/net/slipcor/pvparena/goals/GoalSabotage.java (hunk 3 above)
Comment (Oruss7) [reply src/net/slipcor/pvparena/goals/GoalSabotage.java:593]: It's when region is protected, the tnt disapear and nothing... so I added it =)

**Pull request merged**